source/file/Importer.js

```diff
@@ -110,12 +110,14 @@ class Importer {
     const minY = Math.min(...tileYs)
     const maxY = Math.max(...tileYs)
     const minX = Math.min(...tiles.map(tile => tile.position.x))
+    const yRange = maxY - minY
     return tiles.map(tile => {
       return {
         id: tile.id,
         position: {
           x: (tile.position.x - minX) + (IMPORT_TILE_MARGINS - LEFT_BIAS),
-          y: ((maxY - minY) - (tile.position.y - minY)) + IMPORT_TILE_MARGINS,
+          y: ((yRange - (tile.position.y - minY)) - (yRange % 2))
```

Comment:
Maybe the variable names are weird, but think of `yRange` _as_ `maxY`—but with a pesky offset in the form of `minY`.

And you are totally right about the redundant arithmetic!